wallet/src/repository/wallet/tests/mod.rs

```diff
@@ -793,7 +752,7 @@ fn test_index_transaction_vtt_created_by_wallet() {
 
     let a_block = factories::BlockInfo::default().create();
     let our_address = wallet.gen_external_address(None).unwrap();
-    let their_address = wallet.gen_external_address(None).unwrap();
+    let their_pkh = PublicKeyHash::from_str("wit1mx47n6zp4y574a3xvjt90kefc7lggvgw0v39mf").unwrap();
```

Comment:
Why is this?

Reply:
I changed it to `factories::pkh()`. The problem before was that the generated `pkh` is derived from the same wallet, so that the `index_transactions` was not working as the test was intended.